<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tarea3</class>
 <widget class="QDialog" name="tarea3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="BitSelect">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>461</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="boton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>211</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Apagar/Encender</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Bit a cambiar</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>73</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Numero a utilizar del arreglo</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
